test/spec/modules/livewrappedBidAdapter_spec.js

```diff
@@ -0,0 +1,835 @@
+
+      let expectedQuery = {
+        auctionId: 'F7557995-65F5-4682-8782-7D5D34D82A8C',
+        publisherId: '26947112-2289-405D-88C1-A7340C57E63E',
+        userId: 'user id',
+        url: 'http://www.domain.com',
+        seats: {'dsp': ['seat 1']},
+        version: '1.1',
+        cookieSupport: true,
+        gdprApplies: false,
+        adRequests: [{
+          adUnitId: '9E153CED-61BC-479E-98DF-24DC0D01BA37',
+          callerAdUnitId: 'panorama_d_1',
+          bidId: '2ffb201a808da7',
+          transactionId: '3D1C8CF7-D288-4D7F-8ADD-97C553056C3D',
+          formats: [{width: 980, height: 240}, {width: 980, height: 120}]
+        }]
+      };
+
+      expect(data).to.deep.equal(expectedQuery);
+    });
+
+    it('should pass no cookie support', function() {
+      sandbox.stub(utils, 'cookiesAreEnabled').callsFake(() => false);
+      sandbox.stub(utils, 'isSafariBrowser').callsFake(() => false);
+      let result = spec.buildRequests(bidderRequest.bids, bidderRequest);
+      let data = JSON.parse(result.data);
+
+      expect(result.url).to.equal('https://lwadm.com/ad');
+
+      let expectedQuery = {
+        auctionId: 'F7557995-65F5-4682-8782-7D5D34D82A8C',
+        publisherId: '26947112-2289-405D-88C1-A7340C57E63E',
+        userId: 'user id',
+        url: 'http://www.domain.com',
+        seats: {'dsp': ['seat 1']},
+        version: '1.1',
+        cookieSupport: false,
+        adRequests: [{
+          adUnitId: '9E153CED-61BC-479E-98DF-24DC0D01BA37',
+          callerAdUnitId: 'panorama_d_1',
+          bidId: '2ffb201a808da7',
+          transactionId: '3D1C8CF7-D288-4D7F-8ADD-97C553056C3D',
+          formats: [{width: 980, height: 240}, {width: 980, height: 120}]
+        }]
+      };
+
+      expect(data).to.deep.equal(expectedQuery);
+    });
+
+    it('should pass no cookie support Safari', function() {
+      sandbox.stub(utils, 'cookiesAreEnabled').callsFake(() => true);
+      sandbox.stub(utils, 'isSafariBrowser').callsFake(() => true);
+      let result = spec.buildRequests(bidderRequest.bids, bidderRequest);
+      let data = JSON.parse(result.data);
+
+      expect(result.url).to.equal('https://lwadm.com/ad');
+
+      let expectedQuery = {
+        auctionId: 'F7557995-65F5-4682-8782-7D5D34D82A8C',
+        publisherId: '26947112-2289-405D-88C1-A7340C57E63E',
+        userId: 'user id',
+        url: 'http://www.domain.com',
+        seats: {'dsp': ['seat 1']},
+        version: '1.1',
+        cookieSupport: false,
+        adRequests: [{
+          adUnitId: '9E153CED-61BC-479E-98DF-24DC0D01BA37',
+          callerAdUnitId: 'panorama_d_1',
+          bidId: '2ffb201a808da7',
+          transactionId: '3D1C8CF7-D288-4D7F-8ADD-97C553056C3D',
+          formats: [{width: 980, height: 240}, {width: 980, height: 120}]
+        }]
+      };
+
+      expect(data).to.deep.equal(expectedQuery);
+    });
+
+    it('should use params.url, then config pageUrl, then bidderRequest.refererInfo.referer', function() {
```

Comment:
This is the only test that was modified, except for changing from "//" to "https://" in all tests against our endpoint url. Can anything in this test cause the error "Some of your tests did a full page reload!"? The other tests that fails are not related to our code changes (openx analytics adapter issues)